src/lib/libstack_trace.js

```diff
@@ -15,8 +15,8 @@ var LibraryStackTrace = {
     // Find the symbols in the callstack that corresponds to the functions that
     // report callstack information, and remove everything up to these from the
     // output.
-    var iThisFunc = callstack.lastIndexOf('_emscripten_log');
-    var iThisFunc2 = callstack.lastIndexOf('_emscripten_get_callstack');
+    var iThisFunc = callstack.lastIndexOf(' _emscripten_log ');
+    var iThisFunc2 = callstack.lastIndexOf(' _emscripten_get_callstack ');
```

Comment:
Is this always correct? Do some engines not have something like `func@src.js:10:20` (with no space after `func`)?

Also, is this testable?

Reply:
You are correct, it was broken in FF.

Updated the test and the PR description accordingly.

Reply:
> Updated the test

Where is that part? I can't see it.

Reply:
Basically I move the check/test for `_emscripten_log` / `_emscripten_get_callstack` down to where we use proper regexes to find the symbols names.